demmt/nvrm_decode_ioctl.c

```diff
@@ -827,12 +827,37 @@ static void decode_nvrm_ioctl_unkd7(struct nvrm_ioctl_unkd7 *s)
 	nvrm_print_ln();
 }
 
+static void decode_nvrm_ioctl_numa_info2(struct nvrm_ioctl_numa_info2 *s)
+{
+	nvrm_print_x32(s, nid);
+	nvrm_print_x32(s, status);
+	nvrm_print_x64(s, memblock_size);
+	nvrm_print_x64(s, numa_mem_addr);
+	nvrm_print_x64(s, numa_mem_size);
+	nvrm_print_ln();
+
+	mmt_log_cont("  %s", "blacklist_addresses: addresses[] ");
+	for (int i = 0; i < 64; i++)
+		mmt_log_cont("0x%016" PRIx64 ", ", s->blacklist_addresses.addresses[i]);
```

Comment:
This line will be quite long... Maybe print only non-zero addresses (with index, so we know which ones are zero)?